apps/intellij/src/main/kotlin/dev/nx/console/generate/NxReMoveProjectDialog.kt

```diff
@@ -0,0 +1,197 @@
+                                        reMoveGeneratorContext?.project
+                                    )
+                                    .apply {
+                                        cell(this)
+                                            .bind(
+                                                { component -> component.text },
+                                                { component, value -> component.text = value },
+                                                model::project.toMutableProperty()
+                                            )
+                                            .comment(getShortcutHint())
+                                            .horizontalAlign(HorizontalAlign.FILL)
+
+                                        addDocumentListener(
+                                            object : BulkAwareDocumentListener.Simple {
+                                                override fun afterDocumentChange(
+                                                    document: Document
+                                                ) {
+                                                    val text = document.text
+                                                    projectOptions?.get(text).let {
+                                                        appsOrLibsText?.text =
+                                                            if (it != null)
+                                                                getDestinationDirHint(it)
+                                                            else ""
+                                                    }
+                                                }
+                                            }
+                                        )
+                                    }
+                        }
+                        .layout(RowLayout.PARENT_GRID)
+                    row {
+                            label("Using:")
+                            if (reMoveGenerators.size > 1) {
+                                comboBox(reMoveGenerators)
+                                    .bindItem(model::generator)
+                                    .horizontalAlign(HorizontalAlign.FILL)
+                            } else {
+                                text(reMoveGenerators.get(0))
+                            }
+                        }
+                        .layout(RowLayout.PARENT_GRID)
+                    row {
+                            label("Destination:")
+                            if (workspaceLayout != null) {
+                                val initialDirHint =
+                                    model.project.let {
+                                        projectOptions?.get(it)?.let { getDestinationDirHint(it) }
+                                    }
+                                        ?: ""
+                                appsOrLibsText = text(initialDirHint).gap(RightGap.SMALL).component
```

Comment:
To always remove the gap
```suggestion
                                initialDirHint?.let {
                                    appsOrLibsText = text(it).gap(RightGap.SMALL).component
                                }
```